src/components/navigation/navigation.jsx

```diff
@@ -292,6 +292,15 @@ var Navigation = React.createClass({
                                         </a>
                                     </li>
                                 ] : []}
+                                {this.state.session.permissions.student ? [
+                                    <li>
+                                        <a href={'/classes/' + this.state.session.user.classroomId + '/'}>
+                                            <FormattedMessage
+                                                id='general.myClass'
+                                                defaultMessage={'My Class'} />
```

Comment:
we're no longer putting default messages inline, and just having them be compiled in the `make translations` step.